src/components/Button/button.stories.tsx

```diff
@@ -107,13 +107,13 @@ export const withSizes = () => (
     <Button colorScheme='blue' size='xs'>
       Button
     </Button>
-    <Button colorScheme='blue' size='sm'>
+    <Button colorScheme='blue' size="small">
```

Comment:
these seem unrelated to the router changes?

Reply:
@0xdef1cafe I believe "sm" to "small" and "lg" to "large"  was a setting of my global typescript.